pkg/gadgets/trace/bind/types/types.go

```diff
@@ -15,20 +15,25 @@
 package types
 
 import (
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	Pid       uint32 `json:"pid,omitempty"`
-	Comm      string `json:"comm,omitempty"`
-	Protocol  string `json:"proto,omitempty"`
-	Addr      string `json:"addr,omitempty"`
-	Port      uint16 `json:"port,omitempty"`
-	Options   string `json:"opts,omitempty"`
-	Interface string `json:"if,omitempty"`
-	MountNsID uint64 `json:"mountnsid,omitempty"`
+	Pid       uint32 `json:"pid,omitempty" column:"pid,width:7,fixed"`
+	Comm      string `json:"comm,omitempty" column:"comm,width:16,fixed"`
+	Protocol  string `json:"proto,omitempty" column:"proto,width:5,fixed"`
```

Comment:
This is how it looks now. The first event was generated with a pod and container named `test-pod`. While for the second, I changed it to `test-pod-long-name-to-test`.

```bash
$ tput cols
95
$ ./kubectl-gadget trace bind -A
NODE       NAMESPACE  POD        CONTAINER PID     COMM             PROTO ADDR  PORT  OPTS  IF
worker     test-ns    test-…test test-pod… 263307  nc               TCP   ::    4242  .R... 0
worker     test-ns    test-pod   test-pod  263789  nc               TCP   ::    4242  .R... 0
```